### DISCUSSION TITLE
Inline code always shows up as the word `children` for me

### DISCUSSION THREAD

**Discussion by kevinb-khan** [mdx-js/mdx#2500] [Q&A]
The following .mdx:
```mdx
<div style={{backgroundColor: 'violet', padding: '1rem'}}>
  Try and change the background color to `tomato`.
</div>
```

gets rendered as:
<img width="380" alt="Screen Shot 2024-06-28 at 1 19 19 PM" src="https://github.com/mdx-js/mdx/assets/173723180/1f63ed1a-4234-4264-b49c-da2c7573ba58">

I tried searching for an existing issue, but couldn't find anything.  As a workaround I can do `<code>tomato</code>`.

My entry point looks like:
```jsx
import React from "react";
import ReactDOM from "react-dom/client";

import Routes from "./routes";

ReactDOM.createRoot(document.getElementById("root")!).render(
    <React.StrictMode>
        <Routes />
    </React.StrictMode>,
);
```

and my vite.config.ts looks like:
```ts
import {defineConfig} from "vite";
import react from "@vitejs/plugin-react-swc";
import mdx from "@mdx-js/rollup";
import remarkGfm from "remark-gfm";

// https://vitejs.dev/config/
export default defineConfig({
    plugins: [mdx({remarkPlugins: [remarkGfm]}), react()],
});
```

I'm using the following mdx related deps:
```json
    "@mdx-js/rollup": "^3.0.1",
    "remark-gfm": "^4.0.0"
```

**Comment by ChristianMurphy** [comment]
Welcome @kevinb-khan! 👋 
Could you share a complete example inside a sandbox?
It's unclear what other versions of tools and what kind of routing is being used, and if those could be influencing the outcome.
https://vite.new/react or https://vite.new/react-ts could be a good starting point for sharing the example.
